### PR TITLE
Create post for release of v.2.8.0

### PR DESCRIPTION
**Pull request by LisaBock**
With this PR a post for the release of v.2.8.0 is added to the website.

We could also send it as email and add it to the GitHub discussions.

### CODE REVIEW

**Review by remi-kazeroni — APPROVED** [review]
Thanks a lot @LisaBock! All looks good to me. You would just need to update the date when that post is published.

File: _posts/2023-03-xx-New-release.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+New versions of the ESMValTool (v2.8.0) and ESMValCore (v2.8.0) have just been released:
+-	https://github.com/ESMValGroup/ESMValTool/releases/tag/v2.8.0
+-	https://github.com/ESMValGroup/ESMValCore/releases/tag/v2.8.0 
+
+These versions are available, as usual, on GitHub, conda-forge, PyPI and as containers. 
+
+Please refer to the Changelogs for an overview of the latest changes and additions:
+-	https://docs.esmvaltool.org/en/latest/changelog.html#v2-8-0
+-	https://docs.esmvaltool.org/projects/esmvalcore/en/latest/changelog.html#v2-8-0
```
Comment (axel-lauer) [inline _posts/2023-03-xx-New-release.md:17]: Would be nice to make the links active so people can click on them.

File: _posts/2023-03-xx-New-release.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: ESMValTool v2.8.0
+date: 2023-03-xx
```
Comment (axel-lauer) [inline _posts/2023-03-xx-New-release.md:4]: "xx" needs to be replace before merging. This also applies to the the filename of this post.

File: _posts/2023-03-xx-New-release.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+These versions are available, as usual, on GitHub, conda-forge, PyPI and as containers. 
+
+Please refer to the Changelogs for an overview of the latest changes and additions:
+-	https://docs.esmvaltool.org/en/latest/changelog.html#v2-8-0
+-	https://docs.esmvaltool.org/projects/esmvalcore/en/latest/changelog.html#v2-8-0 
+
+-------------------
+
+### Highlights:
+ESMValCore:
```
Comment (axel-lauer) [inline _posts/2023-03-xx-New-release.md:22]: Could be marked as headline.

File: _posts/2023-03-xx-New-release.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  on-the-fly to unlock the use of more sophisticated regridding algorithms.
+   
+- The **Python API** has been extended with the addition of three modules: [`esmvalcore.config`](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/api/esmvalcore.config.html#module-esmvalcore.config), [`esmvalcore.dataset`](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/api/esmvalcore.dataset.html#module-esmvalcore.dataset), and [`esmvalcore.local`](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/api/esmvalcore.local.html#module-esmvalcore.local). 
+   
+   For details, see our new [example notebooks](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/example-notebooks.html).
+
+- The preprocessor **[`multi_model_statistics()`](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/recipe/preprocessor.html#multi-model-statistics)**
+  has been extended to support more use-cases.
+   
+ESMValTool:
```
Comment (axel-lauer) [inline _posts/2023-03-xx-New-release.md:41]: Same here, maybe mark as headline.

**Review by axel-lauer — APPROVED** [review]
Please see my suggestions / comments. Other than that, looks all good! Thanks @LisaBock !

**Comment by LisaBock** [comment]
Thanks @remi-kazeroni and @axel-lauer for your review!

I just need to adjust the date when the day comes...

**Pull request merged**